meta-aws-iot-automotive-demo/recipes-core/images/aws-biga-image/aws-biga-image.bb

```diff
@@ -156,3 +156,4 @@ IMAGE_INSTALL += "netcat"
 IMAGE_INSTALL += "curl"
 
 IMAGE_INSTALL += "ntp-systemd"
+
```

Comment:
Is this new line needed?

Reply:
You're right there is no reason for this. In some cases it is just good practice, but I found nothing about this on bitbake style guide etc.
Will remove this file from PR